addons/mail/static/src/components/attachment_viewer/attachment_viewer.xml

```diff
@@ -39,7 +39,7 @@
                                 <i class="fa fa-3x fa-circle-o-notch fa-fw fa-spin" role="img" title="Loading"/>
                             </div>
                         </t>
-                        <img class="o_AttachmentViewer_view o_AttachmentViewer_viewImage" t-on-click="_onClickImage" t-on-mousedown="_onMousedownImage" t-on-wheel="_onWheelImage" t-att-src="attachmentViewer.attachment.defaultSource" t-att-style="imageStyle" draggable="false" alt="Viewer" t-ref="image"/>
+                        <img class="o_AttachmentViewer_view o_AttachmentViewer_viewImage" t-on-click="_onClickImage" t-on-mousedown="_onMousedownImage" t-on-wheel="_onWheelImage" t-on-load="_onLoadImage" t-att-src="attachmentViewer.attachment.defaultSource" t-att-style="imageStyle" draggable="false" alt="Viewer" t-key="'image_' + attachmentViewer.attachment.id" t-ref="image_{{attachmentViewer.attachment.id}}"/>
```

Comment:
It's a bit strange to use `+` to build the key but `{{ }}` to build the ref. Can't we use the same syntax for both?
Note: If you use the syntax of ref, should have space inside { }

Reply:
still mix of syntax here

Reply:
`t-ref` and `t-key` don't work the same way, `t-key` expects a string, `t-ref` expects a formatted expression